integration-tests/testdata/cpp-multimodule-project/sonar-project.properties

```diff
@@ -0,0 +1,56 @@
+sonar.host.url=http://localhost:9000
+
+sonar.modules=cli,lib,package1,package2
+cli.sonar.projectBaseDir=cli
+cli.sonar.sources=.
+cli.sonar.projectName=cli
+cli.sonar.cxx.includeDirectories=../../googletest_project/tests/gtest-1.6.0/include/gtest
+lib.sonar.projectBaseDir=lib
+lib.sonar.sources=.
+lib.sonar.projectName=lib
+package1.sonar.projectBaseDir=package1
+package1.sonar.sources=.
+package1.sonar.projectName=package1
+package2.sonar.projectBaseDir=package2
+package2.sonar.sources=.
+package2.sonar.projectName=package2
+
+sonar.projectKey=cpp-multimodule-project
+sonar.projectName=cpp-multimodule-project
+sonar.projectVersion=Work
+sonar.dotnet.key.generation.strategy=safe
+sonar.scm.enabled=true
```

Comment:
In the other configuration files I set scm to false. Good to see how this is working.